packages/cloud-cognitive/src/components/ComboButton/_combo-button.scss

```diff
@@ -14,52 +14,60 @@
   /// Sets the prefix for all IBM Security-specific classes.
   /// @access private
   /// @type String
-  $security--prefix: security;
+  //$security--prefix: cfp;
```

Comment:
Can we remove this comment?

Reply:
Hmmm that's odd... I switched over to $blockClass. I will take a look at it.